plugins/test-coverage/skills/test-structure/README_HTML_REPORTS.md

```
@@ -0,0 +1,294 @@
+```python
+from html_report_generator import generate_test_structure_html
+```
+
+If you see an ImportError, ensure `html_report_generator.py` is in the same directory.
+
+## Support
+
+For issues or questions:
+- Check the main plugin README: `../../README.md`
+- Review the SKILL.md documentation: `SKILL.md`
+- File an issue: https://github.com/openshift-eng/ai-helpers/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)